Demo/Tests/SnapshotTest/Extensions.swift

```diff
@@ -0,0 +1,91 @@
+//  Copyright © 2017 One by AOL : Publishers. All rights reserved.
+
+extension UITraitCollection {
+    enum iPhone {
+        enum iPhoneRegular {
+            static let portrait = UITraitCollection(
+                traitsFrom: [UITraitCollection(userInterfaceIdiom: .phone),
+                             UITraitCollection(verticalSizeClass: .regular),
+                             UITraitCollection(horizontalSizeClass: .compact)])
+            static let landscape = UITraitCollection(
+                traitsFrom: [UITraitCollection(userInterfaceIdiom: .phone),
+                             UITraitCollection(verticalSizeClass: .compact),
+                             UITraitCollection(horizontalSizeClass: .compact)])
+        }
+        enum iPhonePlus {
+            static let portrait = UITraitCollection(
+                traitsFrom: [UITraitCollection(userInterfaceIdiom: .phone),
+                             UITraitCollection(verticalSizeClass: .regular),
+                             UITraitCollection(horizontalSizeClass: .compact)])
+            static let landscape = UITraitCollection(
+                traitsFrom: [UITraitCollection(userInterfaceIdiom: .phone),
+                             UITraitCollection(verticalSizeClass: .compact),
+                             UITraitCollection(horizontalSizeClass: .regular)])
+        }
+    }
+    enum iPad {
+        static let fullScreen = UITraitCollection(
+            traitsFrom: [UITraitCollection(userInterfaceIdiom: .pad),
+                         UITraitCollection(horizontalSizeClass: .regular),
+                         UITraitCollection(verticalSizeClass: .regular),])
+        static let splitOneThird = UITraitCollection(
+            traitsFrom: [UITraitCollection(userInterfaceIdiom: .pad),
+                         UITraitCollection(horizontalSizeClass: .regular),
+                         UITraitCollection(verticalSizeClass: .compact),])
+        static let splitTwoThirds = UITraitCollection(
```

Comment:
`splitTwoThirds` == `fullScreen` ?

Reply:
Yes, full screen and 2/3 on iPad have the same size classes, but it was planned this way so that when someone goes through the hierarchy of Presentation, he would have a clear understanding of what is going on.
Otherwise, trait collections like fullScreen, 2/3 and half could be a single TraitCollection named "regular", for example. It is being used only in HostWindow and would be used again only if we add new device, so it makes sense.

Reply:
Ok